### ISSUE TITLE
Issue but maybe not an issue

### ISSUE THREAD

**Issue by ermurenz** [OpenKMIP/PyKMIP#661]
Hi guys,
my english sucks so i apologize in advance.
This is my experience.

I installed PyKMIP on our vCenter VMware with v6.7 an it work perfectly on any aspect.
All clean on both sides (appliance and vcenter).
We tried with a vCenter VMware 7.0.2.00400 and it seems to work fine: encryption and decryption works and vcenter not show any issue.
Noerror on vCenter console.
Just out of curiosity I went to check the /var/log/pykmip/server.log
on the PyKMIP side, and it show this type of log:

2022-02-13 02:07:36,575 - kmip.server.engine - INFO - Processing operation: GetAttributes
2022-02-13 02:07:36,583 - kmip.server.session.00000040 - WARNING - Failure parsing request message.
2022-02-13 02:07:36,583 - kmip.server.session.00000040 - ERROR -
Traceback (most recent call last):
File "/usr/local/lib/python3.8/dist-packages/PyKMIP-0.11.0.dev1-py3.8.egg/kmip/services/server/session.py", line 167, in _handle_message_loop
request.read(request_data, kmip_version=kmip_version)
File "/usr/local/lib/python3.8/dist-packages/PyKMIP-0.11.0.dev1-py3.8.egg/kmip/core/messages/messages.py", line 485, in read
batch_item.read(istream, kmip_version=kmip_version)
File "/usr/local/lib/python3.8/dist-packages/PyKMIP-0.11.0.dev1-py3.8.egg/kmip/core/messages/messages.py", line 307, in read
self.request_payload = self.payload_factory.create(
File "/usr/local/lib/python3.8/dist-packages/PyKMIP-0.11.0.dev1-py3.8.egg/kmip/core/factories/payloads/init.py", line 48, in create
return self._create_add_attribute_payload()
File "/usr/local/lib/python3.8/dist-packages/PyKMIP-0.11.0.dev1-py3.8.egg/kmip/core/factories/payloads/init.py", line 147, in _create_add_attribute_payload
raise NotImplementedError()
NotImplementedError
2022-02-13 02:07:36,585 - kmip.server.session.00000040 - INFO - Stopping session: 00000040

This thing happens only at the 99% of the encryption procedure of the virtual machines:
Literally on the last second , but every time.
No problem on decryptions.
Again, beside  this error  everything seems to work correctly, the vms are encrypted and decrypted without problems on the vmware side.
All "actions" on the encrypted vms goes fine.
No error showed.
In short, no problem except that particular log.
Doesn't seem the case but, could it be a symptom of some worse stuff?

I hope thit is the right section for this type of question.If not, i apologise.

any advise is appreciate.

**Comment by arp102** [comment]
You're seeing that error because the "Add Attribute" operation is not currently supported in PyKMIP.
You may want to find out how "Add Attribute" is being to used to see if it's important for your use case.

PyKMIP does support the "Set Attribute" and "Modify Attribute" operations, which have basically equivalent functionality, so those operations could potentially be used as a workaround.
